vnext/PropertySheets/FixupRoslynCscWarnings.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <!--
+    ARM64 C++ projects will by default add $(DotNetSdk_LibraryPath_arm64) and
+    $(VC_LibraryPath_ATL_ARM64) to the LIB environment variable, even if the
+    paths do not exist. This causes compiler warnings from the csc instance
+    spawned by RoslynCodeTaskFactory in our inline tasks.
+
+    This target alters the build environment to prevent the warning from the
+    RoslynCodeTaskFactory assembly spawned csc process.
+  -->
+  <Target 
+    Name="ReactNativeWindowsFixupRoslynCscWarnings"
+    Condition="($(LIB.Contains('$(DotNetSdk_LibraryPath_arm64)')) and !Exists('$(DotNetSdk_LibraryPath_arm64)')) or ($(LIB.Contains('$(VC_LibraryPath_ATL_ARM64)')) and !Exists('$(VC_LibraryPath_ATL_ARM64)'))"
+    AfterTargets="SetBuildDefaultEnvironmentVariables"
+    BeforeTargets="PrepareForBuild">
+    <SetEnv Name="LIB" Value="$(LIB.Replace('$(DotNetSdk_LibraryPath_arm64)', '').Replace('$(VC_LibraryPath_ATL_ARM64)', ''))" Prefix="False" />
```

Comment:
Note that we cannot do something like adding `ItemDefinitionGroup` for csc items to disable the warning, since the `csc` process is spawned by a separate assembly.

Reply:
We could I think supress it at an MSBuild invocation level for anywhere we have warnaserror, but that wouldn't really help external projects.

Reply:
I can't see to pass warning suppression via env variable either...